### PR TITLE
Feat(solidity): add solidity contracts package

### PR DESCRIPTION
**Pull request by guidiaz**
- First approach.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,5 +23,9 @@
   "volta": {
     "node": "14.17.6",
     "yarn": "1.22.10"
+  },
+  "dependencies": {
+    "@openzeppelin/contracts": "^4.3.2",
+    "witnet-ethereum-bridge": "~0.4.6"
```
Comment (mariocao) [inline package.json:29]: I think those packages should be in the `core` package

File: packages/core/build/contracts/IERC165.json [hunk 2]
```diff
@@ -0,0 +1,334 @@
+{
```
Comment (mariocao) [inline packages/core/build/contracts/IERC165.json:1]: Build artifacts are usually not committed :)

File: .gitignore [hunk 3]
```diff
@@ -71,3 +72,7 @@ stickers/qr_codes
 
 # python deps
 stickers/src
+
+# package lock files
+package-lock.json
+yarn.lock
```
Comment (mariocao) [inline .gitignore:78]: Lock files are usually included to ensure that the project builds correctly

File: packages/core/build/contracts/IERC165.json (hunk 2 above)
Comment (Tommytrg) [reply packages/core/build/contracts/IERC165.json:1]: To avoid commit `/build`, add `"prepare": "you_build_script"` in `package.json` scripts

Comment (guidiaz) [reply packages/core/build/contracts/IERC165.json:1]: forgot to add it to packages/code/.gitignore

**Review by mariocao — APPROVED** [review]

**Pull request merged**